temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
+        else:
+            log.error("ERROR: Invalid file type")
```

Comment:
**Suggestion:** Handle the case where the file does not exist or is not accessible. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
import json
import os

def reset_files(filepath, file_type="txt"):
    if not os.path.exists(filepath):
        log.error(f"ERROR: File {filepath} does not exist")
        return
    try:
        with open(filepath, "w") as output_file:
            if file_type == "txt":
                output_file.write("")
            elif file_type == "json":
                output_file.write(json.dumps({}))
            else:
                log.error("ERROR: Invalid file type")
    except PermissionError:
        log.error(f"ERROR: No permission to write to file {filepath}")
```